konig-etl/src/main/java/io/konig/etl/aws/PrepareToExport.java

```diff
@@ -30,10 +30,11 @@ public void process(Exchange exchange) throws Exception {
     	String targetTable = exchange.getIn().getHeader("targetTable", String.class);
     	String bucketName = exchange.getIn().getHeader("bucketName", String.class);
     	String fileName = exchange.getIn().getHeader("fileName", String.class); 
-        exchange.getOut().setBody("SELECT * FROM "+targetTable +" INTO OUTFILE S3 's3://"+bucketName+"/"+fileName+"' "
+    	String modified = exchange.getIn().getHeader("modified", String.class); 
+        exchange.getOut().setBody("SELECT * FROM "+targetTable +" WHERE modified='"+modified+"' INTO OUTFILE S3 's3://"+bucketName+"/"+fileName+"' "
```

Comment:
I think you need to wrap the `modified` value within a `TO_TIMESTAMP` function to convert the data type from String to Timestamp.